frontend/src/__mocks__/mockRegisteredModel.ts

```diff
@@ -4,13 +4,17 @@ import { createModelRegistryLabelsObject } from './utils';
 type MockRegisteredModelType = {
   id?: string;
   name?: string;
+  author?: string;
+  owner?: string;
```

Comment:
we should only have `owner` here, not `author`. Same for the other two instances below in this file.